code/modules/mob/living/simple_mob/subtypes/animal/alien animals/succlet.dm

```diff
@@ -0,0 +1,197 @@
+/datum/category_item/catalogue/fauna/succlet
+	name = "Alien Wildlife - Succlet"
+	desc = "The Catslug is an omnivorous terrestrial creature.\
+	Exhibiting properties of both a cat and a slug (hence its name)\
+	it moves somewhat awkwardly. However, the unique qualities of\
+	its body make it exceedingly flexible and smooth, allowing it to\
+	wiggle into and move effectively in even extremely tight spaces.\
+	Additionally, it has surprisingly capable hands, and moves quite\
+	well on two legs or four. Caution is advised when interacting\
+	with these creatures, they are quite intelligent, and proficient\
+	tool users."
+	value = CATALOGUER_REWARD_MEDIUM
+
```

Comment:
The cataloguer description for the succlet refers to catslugs, rather than the succlet.